Public/New-HTMLTab.ps1

```diff
@@ -90,6 +91,14 @@ function New-HTMLTab {
     if ($TextColor) {
         $StyleText.'color' = ConvertFrom-Color -Color $TextColor
     }
+    # New-HTMLTab - Add text-transform
+    if($TextTransform){
+        $StyleText.'text-transform' = "$TextTransform"
+    }
+    else{
+        $StyleText.'text-transform' = "uppercase"
+    }
+    # end
```

Comment:
Removed the if/else statement and left `$StyleText.'text-transform' = "$TextTransform"`

Reply:
You only reviewed, but didn't submit any code changes :-)

Reply:
I'm sorry, I'm new with GitHub. Should I attach the modified code or wait for the Request Changes?

Reply:
you can simply modify code and push code changes. Until you actually change branch it should keep going into this PR.